meetings/eng/2021.md

```diff
@@ -1,4 +1,96 @@
+            - DA: One way you could do this is to give the support steward the ability to _assign_ others to help them with items.
+            - TODO: we'll try the approach in the proposal and see how it goes, make changes as needed
+            - TODO: define an official process that the /support repository is where the support tickets come into
+
+- How to incorporate issue / deliverables review into this meeting? (10-20m) (07:54)
+    - We've discussed using this meeting to review things that we're working on, prioritize, etc
+    - Is there a way we can incorporate this into the team meeting?
+    - Sprint meetings, working through the project board, limit to 30mins
+    - Will conflating a team discussion with sprint meetings eat into each other's time? But also, we're in different time zones and arranging another meeting is overhead
+    - DA agrees with SG that a dedicated sprint meeting separate to team discussion meeting might be best
+    - TODO: set up a separate meeting time slot. maybe every 2 weeks and do the support steward swap as well.
+    - CH: is it OK if people skip every other meeting because of timezones?
+        - YP: intuition is that it's better to have a single meeting group rather than two groups of meetings
+        - CH: is this time sustainable? (Yuvi: Yes, if I don't return to CA :P)
+
+- Discuss possible interaction with JupyterHub team meeting & Pangeo meetings (08:10)
+    - We want to avoid having "community drift" between the Pangeo / JupyterHub meetings
+    - We should make sure that we are systematically publicizing the work we're doing across these meetings
+    - We need intentional processes around information sharing between different groups
+    - TODO: Bake this information into an "open source strategy" for 2i2c.
+        - Information share between upstream projects that are related
+        - Make sure we don't speak "on behalf" of upstream projects - we can have wishes but ultimately need to work with the community
+        - Keep track of our "upstream" work items (e.g., in /external)
+        
+
+- Discuss a new opportunity from Chelle @ NASA
+    - https://github.com/2i2c-org/meta/issues/216
+    - YP: thinks it is super awesome, but also a ton of work
+        - Tons of bureaucracy and red tape
+        - TODO: figure out how many of the challenges here are technical vs. administrative
+        - TODO: Consider having a dedicated person to do the wrangling / coordination / etc
+    - DA: Good opportunity to be at the center of something awesome, but also a ton of effort
+    - ES: Excited about a broader scope of open source development we could  in a somewhat sustainable way from 2i2c via this, but indeed it would be complicated and would needs a dedicated person working on it.
+
+- Team Photo? :joy: 
+    - TODO: chris should put this photo somewhere
+    - Oh no, Chris couldn't find the photo anywhere on his computer! Somehow it was not taken :-(
+
```

Comment:
```suggestion

### Skipped/Backlog

```

Wonder if we should explicitly define this section as containing the agenda points we didn't get to and use it to populate the agenda for the next meeting?

Reply:
Sounds good to me!

Reply:
ah that's a cool idea - I don't know that we want to explicitly commit to covering these topics in the next meeting (IME sometimes people throw out some ideas to discuss but they aren't insistent on discussing it if we don't get to it), but I think it's a great idea to make this explicit as it could be helpful for creating future agendas, for sure!